webrtc.html

```diff
@@ -9504,7 +9513,39 @@ <h3>RTCPeerConnection Interface Extensions</h3>
       <div>
         <pre class="idl">partial interface RTCPeerConnection {
     Promise&lt;RTCStatsReport&gt; getStats (optional MediaStreamTrack? selector = null);
-};</pre>
+    attribute EventHandler onstatslifetimeended;
+          };</pre>
+        <section>
+          <h2>Attributes</h2>
+          <dl data-link-for="RTCPeerConnection"
+              data-dfn-for="RTCPeerConnection" class="attributes">
+            <dt><dfn><code>onstatslifetimeended</code></dfn> of type
+              <span class="idlAttrType"><a>EventHandler</a></span></dt>
+            <dd>
+              <p>The event type of this event handler
+                is <code><a>statslifetimeended</a></code>.
+              </p>
+              <p>When the lifetime of one or more monitored objects ends, the UA MUST
+                queue a task to run the following steps:
+                <ul>
+                  <li>Create a new RTCStatsReport object called <var>r</var></li>
+                  <li>For each monitored object, create a new RTCStats
+                    object with the statatistics for that object, and
+                    add it to <var>r</var>
+                  </li>
+                  <li>
+                    Create a new <a>RTCStatsLifetimeEndedEvent</a>
+                    called <var>e</var>, with its name set to
+                    "statsended" and its "report" being set to <var>r</var>
```

Comment:
Is the event named "statsended" or "statslifetimeended"?

Reply:
What do you think it should be named? @jan-ivar suggested "statsended", but it seems that I didn't change it everywhere.

Reply:
Both sound fine to me, I was just noting the inconsistency